### PR TITLE
Adds Easter Holidays for cfd-oanda and cfd-fxcm in market-hours-database.json

### PR DESCRIPTION
**Pull request by AnshulYADAV007**

### CODE REVIEW

File: Data/market-hours/market-hours-database.json [hunk 1]
```diff
@@ -5789,7 +5873,8 @@
         "1/30/2017",
         "2/16/2018",
         "2/5/2019",
-        "1/27/2020"
+        "1/27/2020",
+        "14/04/2017"
```
Comment (StefanoRaggi) [inline Data/market-hours/market-hours-database.json]: Holiday dates are should all be in M/d/yyyy format, not dd/MM/yyyy.

Comment (AnshulYADAV007) [inline Data/market-hours/market-hours-database.json]: I am so sorry. I have fixed it. Thanks, Stefano.

Comment (StefanoRaggi) [inline Data/market-hours/market-hours-database.json]: No worries, maybe in the future we'll change it to a more international standard format, such as  [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) (yyyy-MM-dd)

**Comment by jaredbroad** [comment]
Hi @AnshulYADAV007; please avoid making white space changes. These make it difficult for others to merge in as it increases the chance of a conflict. The only changes in the file should be ones that you've made directly.

**Pull request closed without merging**